docker/Dockerfile

```diff
@@ -4,10 +4,10 @@ ARG REGISTRY=${REGISTRY:-registry.mapotempo.com/}
 ARG VROOM_VERSION=${VROOM_VERSION}
 
 # Install Vroom
-FROM ${REGISTRY}mapotempo/vroom:${VROOM_VERSION} as vroom
+FROM ${REGISTRY}mapotempo/vroom:${VROOM_VERSION:-v1.5.0} as vroom
```

Comment:
This will depend on https://github.com/Mapotempo/optimizer-api/pull/107 integration

Reply:
#107 uses the new functionality but we can do this without #107 for the existing Vroom functionality. Actually it was already the case in Travis.yml -- that is everything is tested with 1.5.0 since quite some time -- and 1.5.0 is validated for existing functionality.

Reply:
This incoming PR will change the version. The Important thing is that it's allocated here.